tests/tests.c

```diff
@@ -1828,7 +1828,7 @@ bool sirtimerstart(sir_timer* timer) {
 #if !defined(__WIN__)
     int gettime = clock_gettime(SIRTEST_CLOCK, &timer->ts);
     if (0 != gettime) {
-        handle_os_error(true, "clock_gettime(%d) failed!", SIRTEST_CLOCK);
+        handle_os_error(true, "clock_gettime(%lld) failed!", CLOCK_CAST SIRTEST_CLOCK);
```

Comment:
I believe what you meant to do here was on AIX, cast to int, and leave the format statement as it was :)

Reply:
duh :)